src/v/security/scram_algorithm.h

```diff
@@ -68,6 +73,10 @@ class client_first_message {
         return ssx::sformat("n={},r={}", _username, _nonce);
     }
 
+    ss::sstring message() const {
+        return ssx::sformat("n,{},{}", _authzid, bare_message());
```

Comment:
can you make a code comment about this 3 comas and the format. otheriwise what's `n`

Reply:
the file is already commented and references https://tools.ietf.org/html/rfc5802. this is clear from context for anyone reading the file.

Reply:
i see. ok. i was only reading the change.

Reply:
ahh. yes, in that case it would be weird and your comment is justified :)